README.md

```diff
@@ -36,8 +41,70 @@ let countryData = countriesLib.getCountryDataBySlug('united-kingdom');
 let countryData = countriesLib.getCountryByDisplayName('United Kingdom');
 ```
 
+### Making country data more usable
```

Comment:
Maybe we could utilise https://www.npmjs.com/package/documentation instead of documenting this by hand?